packages/spanner/system-test/spanner.js

```diff
@@ -2716,7 +2716,7 @@ var spanner = new Spanner(env);
           transaction.run('SELECT * FROM TxnTable', function(err, rows) {
             assert.ifError(err);
 
-            assert.strictEqual(rows.length, 2);
+            assert.strictEqual(rows.length, 1);
```

Comment:
@stephenplusplus this was my fault. So when we capture the timestamp from inserts, they're approximate, in that we capture them before an insert happens, meaning when we grab the timestamp from insert `n`, it should return `n - 1` values.